vue-frontend/src/components/home-new/LandingSection.vue

```diff
@@ -380,4 +380,10 @@ export default {
     line-height: 5.875rem;
   }
 }
+
+@supports (-webkit-touch-callout: none) {
+  .v2-header-3-text {
+    letter-spacing: -0px;
```

Comment:
What is the different between 0px and -0px ?? Is it valid css property value ??

Reply:
changed.